assets/js/blocks/featured-product/block.js

```diff
@@ -212,7 +225,40 @@ class FeaturedProduct extends Component {
 		);
 	}
 
+	renderApiError() {
+		const { error } = this.state;
+		const onRetry = () => {
+			error.retrying = true;
+			this.setState( { error: error } );
+			error.retry();
+		};
+		return (
+			<Placeholder
+				icon={ <Gridicon icon="notice" /> }
+				label={ __( 'Sorry, an error occured' ) }
```

Comment:
Missing translation domain? Also, is there a typo in _occur**r**ed_?
```suggestion
				label={ __( 'Sorry, an error occurred', 'woo-gutenberg-products-block' ) }
```